pkg/api/openshiftcluster.go

```diff
@@ -230,6 +232,25 @@ type ServicePrincipalProfile struct {
 	SPObjectID   string       `json:"spObjectId,omitempty"`
 }
 
+// PlatformWorkloadIdentityProfile encapsulates all information that is specific to workload identity clusters.
+type PlatformWorkloadIdentityProfile struct {
```

Comment:
While you're in here, it might help to update this file with the remainder of the new fields we added: https://github.com/Azure/ARO-RP/blob/master/pkg/api/openshiftcluster.go#L794-L811 we'll need them later.